tests/e2e/utils/PreferencesHandler.ts

```diff
@@ -57,18 +57,31 @@ export class PreferencesHandler {
 
     private async setPreference(attribute: string, value: string) {
         Logger.trace(`PreferencesHandler.setPreferences ${attribute} to ${value}`);
-        const response = await this.requestHandler.get('api/preferences');
+        var response;
+        try {
+            response = await this.requestHandler.get('api/preferences');
+        } catch (e) {
+            Logger.error(`PreferencesHandler.setPreferences failed to get user preferences`);
+            throw e;
+        }
         const userPref = response.data;
         try {
-            const theiaPref = JSON.parse(userPref['theia-user-preferences']);
+            var theiaPref = JSON.parse(userPref['theia-user-preferences']);
```

Comment:
Is there a reason why are you changing `const` to `var` here?

Reply:
Because the theiaPref object is being edited, I think that's what was causing issues with the preferences not being updated, or am I wrong?

Reply:
Ah, yeah, I missed that it is updated. Good catch - but please use `let` instead of `var` if possible. :)

Reply:
fixed.